content/dart/concepts/switch/operators/operators.md

```diff
@@ -0,0 +1,128 @@
+| **Arithmetic Operators**             | `+`      | Addition                                   |
+|                                      | `-`      | Subtraction                                |
+|                                      | `*`      | Multiplication                             |
+|                                      | `/`      | Division                                   |
+|                                      | `%`      | Modulus                                    |
+|                                      | `~/`     | Truncating Division                        |
+| **Equality and Relational Operators**| `==`     | Equal                                      |
+|                                      | `!=`     | Not Equal                                  |
+|                                      | `>`      | Greater Than                               |
+|                                      | `<`      | Less Than                                  |
+|                                      | `>=`     | Greater Than or Equal To                   |
+|                                      | `<=`     | Less Than or Equal To                      |
+| **Type Test Operators**              | `is`     | True if the object has the specified type  |
+|                                      | `is!`    | True if the object does not have the specified type |
+| **Logical Operators**                | `&&`     | Logical AND                                |
+|                                      | `||`     | Logical OR                                 |
+|                                      | `!`      | Logical NOT                                |
+| **Bitwise Operators**                | `&`      | Bitwise AND                                |
+|                                      | `|`      | Bitwise OR                                 |
+|                                      | `^`      | Bitwise XOR                                |
+|                                      | `~`      | Bitwise NOT                                |
+|                                      | `<<`     | Left shift                                 |
+|                                      | `>>`     | Right shift                                |
+| **Assignment Operators**             | `=`      | Simple Assignment                          |
+|                                      | `+=`     | Add and Assign                             |
+|                                      | `-=`     | Subtract and Assign                        |
+|                                      | `*=`     | Multiply and Assign                        |
+|                                      | `/=`     | Divide and Assign                          |
+|                                      | `%=`     | Modulus and Assign                         |
+|                                      | `~/=`    | Truncating Divide and Assign               |
+
```

Comment:
The information for Logical OR and Bitwise OR is not appearing properly. Please view the file or check it out with a markdown viewer, you'll be able to see it then. Can you fix that? It's probably because of the specific symbol.